client/client-multi/client-cpp/impl/KaaClientProperties.cpp

```diff
@@ -25,9 +25,9 @@
 namespace kaa {
 
 #if __GNUC__
-    static const char FILE_SEPARATOR = '/';
+    static const char *FILE_SEPARATOR = "/";
```

Comment:
Let's move this property to `KaaClientProperties` and use `std::string` instead of `char*`